src/server.h

```diff
@@ -348,6 +348,7 @@ class InferenceServer {
   std::unique_ptr<ModelRepositoryManager> model_repository_manager_;
   std::shared_ptr<TritonBackendManager> backend_manager_;
   std::shared_ptr<TritonCacheManager> cache_manager_;
+  std::shared_ptr<PersistentBackendManager> persist_backend_manager_;
```

Comment:
Remove? I don't think it builds..

Reply:
Not sure why it compiled locally. Removed.